src/js-marketing-notification.ts

```diff
@@ -1,32 +1,31 @@
 import { LocalStorageManager } from './github/gtmsportswear/js-local-storage-manager@1.0.2/local-storage-manager';
 
 export interface Notification {
-  tabContent: string,
-  heading: string;
-  buttonTitle: string;
-  image?: string;
-  content?: string;
+  tabContent: Element,
```

Comment:
Why is `tabContent` an element? Shouldn't we just accept a simple string here?

Also, could we rename this to something that is more representative, such as `toggleText` or `tabText`